pkg/gui/custom_commands.go

```diff
@@ -151,10 +156,78 @@ func (gui *Gui) handleCustomCommandKeybinding(customCommand config.CustomCommand
 						return gui.surfaceError(err)
 					}
 
+					return gui.createMenu(title, menuItems, createMenuOptions{showCancel: true})
+				}
+			case "menuFromCommand":
+				f = func() error {
+					// Collect cmd to run from config
+					cmdStr, err := gui.resolveTemplate(prompt.Command, promptResponses)
+					if err != nil {
+						return gui.surfaceError(err)
+					}
+
+					// Collect Filter regexp
+					filter, err := gui.resolveTemplate(prompt.Filter, promptResponses)
+					if err != nil {
+						return gui.surfaceError(err)
+					}
+					reg, err := regexp.Compile(filter)
+					if err != nil {
+						return gui.surfaceError(errors.New("unable to parse filter regex, error: " + err.Error()))
+					}
+
+					// Run and save output
+					message, err := gui.GitCommand.RunCommandWithOutput(cmdStr)
+					if err != nil {
+						return gui.surfaceError(err)
+					}
+
+					// Need to make a menu out of what the cmd has displayed
+					candidates := []string{}
+					buff := bytes.NewBuffer(nil)
+					temp := template.Must(template.New("format").Parse(prompt.Format))
```

Comment:
Can you remove the `template.Must` wrapper? Just like the filter this is user specific and so could cause errors